### PR TITLE
DCRoom Twig

### PR DESCRIPTION
**Pull request by cconard96**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -

### CODE REVIEW

File: templates/generic_show_form.html.twig [hunk 1]
```diff
@@ -137,7 +138,7 @@
                      {% set dc_breadcrumbs %}
                         {% if item is usingtrait('Glpi\\Features\\DCBreadcrumb') %}
                            {{ call(item.getType() ~ '::renderDcBreadcrumb', [
-                              item.fields['id']
+                              item.getID
```
Comment (trasher) [inline templates/generic_show_form.html.twig:141]: You should avoid using `getID`, its returns non numeric value for new items:
```
An exception has been thrown during the rendering of a template ("DCRoom::renderDcBreadcrumb(): Argument #1 ($items_id) must be of type int, string given").
```

Comment (trasher) [reply templates/generic_show_form.html.twig:141]: (well, using `item.fields['id']` does not solve the error (I've not been able to get that foxed on my side), but anyway using getID is unfortunately not a good idea)

Comment (trasher) [reply templates/generic_show_form.html.twig:141]: Hum... OK, indeed; either I hd another method in mind, either this has changed. I guess error was from an old cache; I no longer see it

**Review by trasher — APPROVED** [review]
I'm not sure all is working as expected - or I have an issue in my instance/db; but it's same in main.

**Review by cedric-anne — APPROVED** [review]

**Pull request merged**